<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementsMainWidget</class>
 <widget class="QWidget" name="ElementsMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>1</number>
     </property>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Period</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="periodSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>  Epsilon/kB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="epsilonDoubleSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>K</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Group</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="groupSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QDoubleSpinBox" name="covalentRadiusDoubleSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Å</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="numberSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,5,2">
       <item>
        <widget class="QLabel" name="rowNumberLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="forceFieldStringIdentifierLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="text">
          <string>H</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SelectColorButton" name="colorPushButton">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Element</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="elementLineEdit">
         <property name="maximumSize">
          <size>
           <width>92</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QDoubleSpinBox" name="vanderWaalsDoubleSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Å</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="oxidationLineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Van der Waals radius  </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>  Triple bond covalent radius  </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Covalent radius  </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Oxidation states</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Atomic radius  </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Atomic mass  </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>User-defined radius  </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QDoubleSpinBox" name="atomicMassDoubleSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Å</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>Sigma</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sigmaDoubleSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>Å</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QDoubleSpinBox" name="atomicRadiusDoubleSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Å</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QDoubleSpinBox" name="tripleBondCovalentRadiusDoubleSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Å</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QDoubleSpinBox" name="userDefinedRadiusDoubleSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Å</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectColorButton</class>
   <extends>QPushButton</extends>
   <header>selectcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
